### ISSUE TITLE
Performance of Tokenbridge

### ISSUE THREAD

**Issue by YuqingWang1029** [Neur-IO/ReVQ#2]
Dear authors, thank you for your excellent work and the comparison with our TokenBridge. We would like to clarify that the rFID score of 1.11 reported for TokenBridge was tested on the ImageNet training set to align with the methodology used in the MAR paper. However, on the validation set, the score is 0.53. If your evaluation metrics are tested on the validation set, we would be grateful if you could update this in the paper accordingly. Thank you very much for this issue.

**Comment by zbr17** [comment]
Thank you for your reminder. We will go into the details of TokenBridge to study it further. Thanks again! We'll come to consult you if we have any unclear points.
